packages/server/lib/browsers/utils.coffee

```diff
@@ -85,18 +87,23 @@ module.exports = {
   getBrowsers: ->
     ## TODO: accept an options object which
     ## turns off getting electron browser?
```

Comment:
The Electron browser can be removed, right? So this `TODO` can be struck from the code?